sassc.c

```diff
@@ -202,6 +202,7 @@ void print_usage(char* argv0) {
     printf("   -m, --sourcemap         Emit source map.\n");
     printf("   -M, --omit-map-comment  Omits the source map url comment.\n");
     printf("   -p, --precision         Set the precision for numbers.\n");
+    printf("   -a, --sass              Treat input as indented syntax.\n");
```

Comment:
to be honnest I didn't even know how to avoid the short syntax (-a). I would have been fine with only `--sass` only, but ended up using `-a` as it wasn't conflicting and I didn't find anything better.